live-examples/js-examples/date/date-tolocaledatestring.js

```diff
@@ -1,7 +1,8 @@
 const event = new Date(Date.UTC(2012, 11, 20, 3, 0, 0));
-
 const options = { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' };
 
+//NOTE: The "expected output" depends on timezone, and may not be correct on your browser
```

Comment:
"not be correct" is a bit strong.

```suggestion
//NOTE: Depending on your local timezone, your output may differ from "expected output"
```